helm_deploy/hmpps-interventions-service/reports/crs_performance_report.sql

```diff
@@ -14,6 +14,25 @@ COPY (
         join action_plan_session aps on apsa.action_plan_session_id = aps.id
       where attended IS NOT NULL
       group by aps.action_plan_id
+  ),
+  first_submitted_action_plans AS (
+    select referral_id, min(submitted_at) as submitted_at
+    from action_plan
+    group by referral_id
+  ),
+  first_approved_action_plans AS (
+     select referral_id, min(approved_at) as approved_at
+     from action_plan
+     group by referral_id
+  ),
```

Comment:
The two above could be combined into a single statement returning both columns from a performance viewpoint